### DISCUSSION TITLE
Chart version

### DISCUSSION THREAD

**Discussion by facorazza** [seaweedfs/seaweedfs-csi-driver#166] [Q&A] [answered]
Why is the chart version 0.2.1 using helm while the release on github is 1.1.9?

**Comment by chrislusf** [comment, marked as answer]
do they need to be the same?

**Reply by facorazza** [reply]
I guess not

**Reply by facorazza** [reply]
would it be possible to release a new version to fix the charts?

**Reply by chrislusf** [reply]
Just need to send a PR? There are no release process.

**Reply by facorazza** [reply]
In the rbac [templates ](https://artifacthub.io/packages/helm/seaweedfs-csi-driver/seaweedfs-csi-driver?modal=template&template=rbac.yaml) the cluster role with name: {{ template "seaweedfs-csi-driver.name". }}-driver-registrar-node-role is missing a `- apiGroups: [""]` on line 172. This is not the case for the chart on [master](https://github.com/seaweedfs/seaweedfs-csi-driver/blob/f1dc976f079bf67edb5d053f8faa4e7427160728/deploy/helm/seaweedfs-csi-driver/templates/rbac.yaml#L172)

**Reply by chrislusf** [reply]
Please send a PR to fix it.

**Reply by facorazza** [reply]
Is there anything else I can do to update the chart version?

**Comment by facorazza** [comment]
The chart on master already contains the fix. I believe that the released version has not been updated yet.

May 29, 2024 19:26:56 Chris Lu ***@***.***>:
